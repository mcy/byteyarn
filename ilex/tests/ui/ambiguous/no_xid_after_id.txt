/foo%q /null%q /foo%qua
