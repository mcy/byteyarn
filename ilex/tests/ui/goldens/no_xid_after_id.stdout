error: unexpected extra characters in ``-, `%q`-suffixed identifier
 --> <input>:1:19
  |
1 | foo%q null%q foo%qua
  |                   ^^ you may have meant to add a space here
  |

error: aborting due to previous error
